### ISSUE TITLE
Math Rendering Issue After MathJax 2.6.1 and SVG Adaption

### ISSUE THREAD

**Issue by dstratossong** [benweet/stackedit#952]
I found an issue with SVG rendering of LaTeX objects - after the most recent commit to adapt SVG rendering. The SVGs are not displaying properly - there appears to be a large space above and left of the SVG, causing it to appear misaligned. 

![screenshot 2016-02-29 at 6 45 04 pm - edited](https://cloud.githubusercontent.com/assets/8782072/13412926/adb44438-df14-11e5-98f0-d5f82db62466.png)

This screenshot shows the outline one of the rendered objects upon selection. The template was generated using the default template, and the HTML could be [found here](https://meetdavidsong.com/test/math-rendering-test.html), and its markdown source is pasted below:
```
Math Rendering Test
===================

Test Inline: $\Sigma_{k=0}^n (k^2+\frac{1}{2}k)$
Test Block: $$\Sigma_1 = \{ a, b, c \}$$ $$\Sigma_2 = \{ \Delta, \text{blue}, 173\}$$
Test Between Text: Let $\Sigma$ be an alphabet, and let $a$, $b$ and $c$ be symbols of $\Sigma$.
```

I've also noticed an error message after MathJax renders all the math objects (that it reached 100%).

> Error preparing HTML-CSS output (postProcess)

This issue was found on the latest Google Chrome, on Chromebook as well as Arch Linux.

However, the math elements renders perfectly fine in the StackEdit preview panel， and I could see that a local version of MathJax with the SVG config is included. Interestingly, changing the MathJax include line to include SVG erases all math elements inside the browser:

```html
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_SVG"></script>
```

I did not see this issue in other MathJax sites, so presumed this might be an issue during the markup process to HTML in StackEdit.

Please take a look at this - as I fully support StackEdit and use it for blogging, note taking everyday!

Regards,
David

**Comment by benweet** [comment]
I think I just found the issue. Should be fixed with v4.3.14.

**Comment by dstratossong** [comment]
Confirmed fixed. You are blazingly fast. Thank you!
I will definitely try to help out more in the future!
